ivy_tests/test_ivy/test_frontends/test_scipy/test_fft/test_scipy_fft.py

```diff
@@ -106,6 +106,36 @@
 #     return dtype, x, type, n, axis, norm
 
 
+# @st.composite
+# def x_and_fft2(draw):
+#     min_fft2_points = 2
+#     dtype = draw(helpers.get_dtypes("float_and_complex", full=False))
+#     x_dim = draw(
+#         helpers.get_shape(
+#             min_dim_size=2, max_dim_size=100, min_num_dims=2, max_num_dims=4
+#         )
+#     )
+#     x = draw(
+#         helpers.array_values(
+#             dtype=dtype[0],
+#             shape=tuple(x_dim),
+#             min_value=-1e5,
+#             max_value=1e5,
+#             allow_inf=False,
+#             large_abs_safety_factor=2.5,
+#             small_abs_safety_factor=2.5,
+#             safety_factor_scale="log",
+#         )
+#     )
+#     s = (
+#         draw(st.integers(min_fft2_points, 256)),
+#         draw(st.integers(min_fft2_points, 256)),
+#     )
+#     dim = draw(st.sampled_from([(0, 1), (-1, -2), (1, 0)]))
+#     norm = draw(st.sampled_from(["backward", "forward", "ortho"]))
```

Comment:
The general format is alright but the test is not general enough because you are hardcoding `dim`. I would suggest you use `dtype_values_axis` to get both `x` and `dim`. In this case you won't need to generate `x_dim` and `dtype` separately either, you can set them using `dtype_values_axis`'s arguments.